File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.js
function solution(n) {
    let numbers = [];
    
    for(let i = 4; i <= n; i++) {
        let cnt = 0;
        for(let j = 1; j <= i; j++) {
            if(i % j === 0) cnt++;
        }
        if(cnt >= 3)
            numbers.push(i)
    }
    
    return numbers.length;
}

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.py
def get_div(n):
    cnt = 1
    for i in range(2, n + 1):
        if not n % i:   cnt += 1
    return cnt

def solution(n):
    return sum([1 for i in range(1, n + 1) if get_div(i) >= 3])